marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/graphEntitiesSettings.cy.tsx

```diff
@@ -335,9 +338,11 @@ describe("Entity Type Settings Modal", () => {
     entitiesSidebar.backToMainSidebar();
 
     cy.log("**Check in the sidebar that the entity type have the new color and icon**");
-    entitiesSidebar.getBaseEntity(defaultEntityTypeData.name).should("be.visible").and("have.attr", "data-color").then(color => {
-      expect(Cypress._.toLower(color)).equal(Cypress._.toLower(newEntityTypeData2.color.HEX));
-    });
+    cy.waitUntil(() =>
```

Comment:
Same as Above

Reply:
@vmendez-marklogic In the video artifact the change that is expected happened a  few moments later of the assertion I think is a valid scenario